src/services/parser.ts

```diff
@@ -14,33 +16,61 @@ addByRSSPodcast: object {
 }
 */
 
-export const getAddByRSSPodcast = async (feedUrl: string) => {
-  const addByRSSPodcastFeedUrlPodcasts = await getAddByRSSPodcasts()
+export const getAddByRSSPodcastLocally = async (feedUrl: string) => {
+  const addByRSSPodcastFeedUrlPodcasts = await getAddByRSSPodcastsLocally()
   return addByRSSPodcastFeedUrlPodcasts.find((x: any) => x.addByRSSPodcastFeedUrl === feedUrl)
 }
 
-export const getAddByRSSPodcasts = async () => {
+export const getAddByRSSPodcastsLocally = async () => {
   try {
     const itemsString = await AsyncStorage.getItem(PV.Keys.ADD_BY_RSS_PODCASTS)
     return itemsString ? JSON.parse(itemsString) : []
   } catch (error) {
-    console.log('getAddByRSSPodcasts', error)
+    console.log('getAddByRSSPodcastsLocally', error)
     return []
   }
 }
 
-const setAddByRSSPodcasts = async (podcasts: any[]) => {
+export const getAddByRSSPodcastFeedUrlsLocally = async () => {
+  try {
+    const itemsString = await AsyncStorage.getItem(PV.Keys.ADD_BY_RSS_PODCAST_FEED_URLS)
+    return itemsString ? JSON.parse(itemsString) : []
+  } catch (error) {
+    console.log('getAddByRSSPodcastFeedUrlsLocally', error)
+    return []
+  }
+}
+
+export const getAddByRSSPodcastFeedUrlsFromServer = async () => {
+  try {
+    const itemsString = await AsyncStorage.getItem(PV.Keys.ADD_BY_RSS_PODCAST_FEED_URLS)
```

Comment:
Is this actually getting the urls from the server?

Reply:
Whooops, no it's not 😬but that's because we don't need it. addByRSSPodcastFeedUrls are now synced with local storage whenever the get-authenticated-user-info endpoint is called.